### PR TITLE
PR #2 

### PR DESCRIPTION
**Pull request by umangS77**
Issues reported on 27/04/21 resolved

### CODE REVIEW

File: experiment/simulation/css/main.css [hunk 1]
```diff
@@ -1 +1,424 @@
...
+    font-size: 2.5vw;
+}
+
+.message {
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.4vw;
+}
+
+.instructionsBox{
```
Comment (raj-vlabs) [inline experiment/simulation/css/main.css]: Please rename this to use a hyphenated name according to the css variable naming convention

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
The issues raised during the meet seem to have been addressed. However, I was just looking at the experiment and noticed the following:
1. For some instructions it is not clear as to which jar/beaker has to be clicked.
2. The mouse pointer doesn't change on the clickable elements. So it is difficult to know which element is clickable and which is not.
3. The click works for all the elements and not only for those required for the next steps. As a result someone can accidentally click a different element and erroneously complete the experiment.
I think the above points need a little bit of attention in the interest of usability.

File: experiment/simulation/css/main.css (hunk 1 above)
Comment (umangS77) [reply experiment/simulation/css/main.css]: Thanks for pointing these issues sir. I'll fix them asap.

File: .github/workflows/deployment-script.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
             node-version: '12'
             check-latest: true
     - run: |
-        git clone -b pipeline-new https://github.com/virtual-labs/ph3-lab-mgmt
+        git clone https://github.com/virtual-labs/ph3-lab-mgmt
```
Comment (raj-vlabs) [inline .github/workflows/deployment-script.yml]: Add --depth=1

File: experiment/references.md [hunk 3]
```diff
@@ -1 +1,12 @@
-### Link your references in here
\ No newline at end of file
+1. Ana Dominguez, Aurora Fernandez, Noemí Gonzalez, Emilia Iglesias, and Luis Montenegro, Journal of Chemical Education, Vol. 74 No. 10 October 1997, pp.1227-1231.  
+2. D. Lopez-Diaz and M. M. Velazquez, The Chemical Educator, Vol. 12, No. 5, 2007, pp. 327-330.    
+3. M. Halder, The Chemical Educator, Vol. 12, No. 1, 2007, (10.1333/ s00897071097a).    
+4. [http://nsdl.niscair.res.in/bitstream/123456789/311/1/PDF+corrected+surface+chemistry.pdf](http://nsdl.niscair.res.in/bitstream/123456789/311/1/PDF+corrected+surface+chemistry.pdf)
```
Comment (raj-vlabs) [inline experiment/references.md]: Change the link to https and provide a meaningful text in place of giving the whole link

**Review by raj-vlabs — APPROVED** [review]
Please remove the http link in references before merging.

**Pull request merged**